docs/modules/getting_started/pages/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Getting started
+
+This guide will get you started with Superset using the Stackable Operator. It will guide you through the installation of the Operator and its dependencies, setting up your first Superset instance, connecting to it and looking at some sample data.
+
+== Prerequisites
+
+You will need:
+
+* a Kubernetes cluster
```

Comment:
see earlier comment: Maybe make a comment here that stackablectl makes this really easy?

Reply:
A mention seems like a good idea!

Reply:
Coming to think of it: I think the `-k` flat isn't *that* big of a helper, because you need docker and kind beforehand. I'd leave it out for now, what I'd rather have is a dedicated documentation page on how to set up a cluster

Reply:
Good point.